benchmark_runner.py

```
@@ -7,78 +7,302 @@
         result_file = RESULTS_DIR / f"{scenario_name}--{model_name}.json"
-        result_file.write_text(json.dumps(result, indent=2))
-        print(f'Results saved to: {result_file}')
+        with open(result_file, 'w') as f:
+            json.dump(result, f, indent=2)
+        
+        # Print summary
+        scores = evaluation_result.get('scores', {})
+        overall_score = scores.get('overall_score', 0)
+        passed = evaluation_result.get('passed', False)
+        
+        print(f"\n✅ Benchmark completed!")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)